cmd/apps/registry_app.go

```diff
@@ -103,9 +106,11 @@ func MakeInstallRegistry() *cobra.Command {
 			return err
 		}
 
+		persistence, _ := registry.Flags().GetBool("persistence")
```

Comment:
Hey @kandros can you catch and check the error that can be returned here please.

Reply:
I agree with this point. Since we are about to do another release, I'll merge and change it myself.

We should check bools in particular for errors.